hy/cmdline.py

```diff
@@ -436,3 +442,10 @@ def _print_for_windows(src):
             print(line)
         except:
             print(line.encode('utf-8'))
+
+# remove PYTHON* environment variables,
+# such as "PYTHONPATH"
+def _remove_python_envs():
+    for key in os.environ.copy().keys():
```

Comment:
```python
for key in list(os.environ.keys()):
    # ...
```
You only need to copy the key view, not the whole dictionary

Reply:
Thanks for suggestion, code has been updated

(I actually used `list` before but for simplicity changed it to something like this, now changing it back :D)